nominees/code-your-future.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Code Your Future",
+  "aliases": [
+    ""
+  ],
+  "description": "A free collaboratively developed open source 8 month web development programme.",
+  "website": "https://codeyourfuture.io/",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://syllabus.codeyourfuture.io/"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceText": "During the course you will learn a technology stack that helps you build anything from a simple website to a powerful database-driven web application.",
+      "evidenceURL": "https://github.com/CodeYourFuture/syllabus"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "content"
+  ],
+  "repositories": [
+    {
+      "name": "syllabus",
+      "url": "https://syllabus.codeyourfuture.io/"
```

Comment:
The content/syllabus is actually at: https://docs.codeyourfuture.io/, and would be great to be able to link to the GitHub repos that source this documentation if possible.

Reply:
Agree I think the links should be:

Digital solution website: https://syllabus.codeyourfuture.io/
Main content: https://syllabus.codeyourfuture.io/Overview
Code + content repo: https://github.com/CodeYourFuture/syllabus